components/connectivity-adapter/internal/connectorservice/model/model.go

```diff
@@ -71,14 +71,46 @@ type TokenResponse struct {
 	Token string `json:"token"`
 }
 
-func MakeCSRURL(newToken, connectivityAdapterBaseURL string) string {
+func MakeCSRInfoResponseWrapper(
```

Comment:
what do you think about `NewCSRInfoResponse` prefix instead of `MakeCSRInfoResponseWrapper `?

Reply:
Yeah, the code is redundant. I will fix it!

Reply:
Sounds cool. I'll change it ^^